OKR: Split up large article into smaller articles (#21626)
dotnet/docs: docs/standard/serialization/system-text-json-invalid-json.md

---

> title: How to allow invalid JSON with System.Text.Json

Reviewer: ```suggestion
title: How to allow some kinds of invalid JSON with System.Text.Json
```

description: "Learn how to allow invalid JSON while serializing to and deserializing from JSON in .NET."
ms.date: 11/18/2020
no-loc: [System.Text.Json, Newtonsoft.Json]
zone_pivot_groups: dotnet-version
helpviewer_keywords:
  - "JSON serialization"
  - "serializing objects"
  - "serialization"
  - "objects, serializing"
---

> # How to allow invalid JSON with System.Text.Json

Reviewer: ```suggestion
# How to allow some kinds of invalid JSON with System.Text.Json
```

> In this article, you will learn how to allow invalid JSON, and write numbers as strings.

Reviewer: ```suggestion
In this article, you will learn how to allow comments, trailing commas, and quoted numbers in JSON, and how to write numbers as strings.
```
